src/content/guides/using-persistent-volumes-with-csi-driver-on-aws/index.md

```diff
@@ -0,0 +1,120 @@
+Alternatively, to be able to set more specific parameters on your PV you can first create a [Persistent Volume object](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes) and then claim that PV using a Persistent Volume Claim (PVC).
+
+Under the hood, the EBS CSI driver will take care that a corresponding EBS Volume with the correct parameters is created.
+
+The EBS Volume and its data will persist as long as the corresponding PV resource exists. Deleting the resource will also delete the corresponding EBS volume, which means that all stored data will be lost at that point.
+
+## Using Persistent Volumes in a Pod
+
+Once you have a Persistent Volume Claim you can [claim it as a Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#claims-as-volumes) in your Pods.
+
+Note that an EBS Volume can only be used by a single Pod at the same time. Thus, the access mode of your PVC can only be `ReadWriteOnce`.
+
+Under the hood the EBS Volume stays detached from your nodes as long as it is not claimed by a Pod. As soon as a Pod claims it, it gets attached to the node that holds the Pod.
+
+## Example
+
+First we create a PVC:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: my-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: ebs-csi
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+Note that the Access Mode while being fixed with EBS still needs to be defined as `ReadWriteOnce` in the manifest.
+
+Further, we need to define the Storage Class `ebs-csi` because it is not the default storage class (default `gp2`).
+
+Now we can create a Pod that uses our PVC:
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: mypod
+spec:
+  containers:
+    - name: myfrontend
+      image: nginx
+      volumeMounts:
+      - mountPath: "/var/www/html"
+        name: mypd
+  volumes:
+    - name: mypd
+      persistentVolumeClaim:
+        claimName: my-claim
+```
+
+Now we have an NGINX Pod which serves the contents of our EBS Volume.
+
+## Expanding Persistent Volume Claims
+
+Starting with Kubernetes v1.16 CSI Volume can be expanded by simply editing the claim and requesting a larger size. It will trigger an update in the underlying Persistent Volume and EBS Volume (Kubernetes always uses the existing one).
+
+Resizing an in-use PersistentVolumeClaim is enabled by default with Kubernetes v1.15, there is no need to delete and recreate a Pod. You can only resize volumes containing a file system if the file system is XFS, Ext3, or Ext4.
+
+__Warning__: Expanding EBS volumes is a time consuming operation. Also, there is a per-volume quota of one modification every 6 hours.
+
+## Deleting Persistent Volumes
+
+The Reclaim Policy is set to `Delete`. Thus, deleting the `PersistentVolume` resource will also delete the respective EBS Volume. Similarly, by default if you delete a `PersistentVolumeClaim` resource the respective Persistent Volume and EBS will get deleted.
```

Comment:
What is the Reclaim Policy? I haven't seen it mentioned so far. Ideally explain it briefly in a sentence before this one.

```suggestion
The Reclaim Policy is set to `Delete`. Thus, deleting the `PersistentVolume` resource will also delete the respective EBS volume. Similarly, by default if you delete a `PersistentVolumeClaim` resource, the respective `PersistentVolume` and EBS volume will get deleted.
```